spektral/datasets/tud.py

```diff
@@ -58,6 +58,8 @@ def load_data(dataset_name, clean=False):
     # Read data
     A_list, X_list, y = _read_graphs(dataset_name)
 
+    print(f"Successfully loaded {dataset_name}")
```

Comment:
This would break compatibility with Python 3.5. 
Can you replace it with `print('Successfully loaded {}.'.format(dataset_name))`?

Cheers